services-validation.md

```diff
@@ -567,7 +567,7 @@ Then in your call to `Validator::make` use the `Lang:get` to use your custom fil
 
 There are a variety of helpful validation rules; however, you may wish to specify some of your own.
 
-The recommended way of adding your own validation rule is to extend the Validator instance via the `extend` method. In an October CMS plugin, this can be added to the `boot()` callback method inside your `Plugin.php` registration file.
+The recommended way of adding your own validation rule is to extend the Validator instance via the `extend` method. In an October CMS plugin, this can be registered either in the `boot()` callback method inside your `Plugin.php` registration file or with the `registerValidationRules` method of the [Plugin registration class](../plugin/registration#registration-methods). The method should return an array where the key is the validator rule name and the value is a callable function. The callable function receives four arguments, the name of the `$attribute` being validated, the `$value` of the attribute and an array of `$parameters` passed to the rule, and the `$validator` instance.
```

Comment:
Could you provide an example of the registerValidationRules() method? Even just the one from the unit test is fine.

Reply:
should be located under the heading that the plugin registration page links to